packages/osd-cross-platform/src/process.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { execSync } from 'child_process';
+
+let workingDir = process.cwd();
+
+if (process.platform === 'win32') {
+  try {
+    const pathFullName = execSync('powershell "(Get-Item -LiteralPath $pwd).FullName"', {
```

Comment:
So does this take care of resolving any short name path set for directories to plugin's full name path?

Reply:
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-item?view=powershell-7.2#-literalpath

.FullName is literally same as `realpath` on linux.


Conbine both you get the actual full path no short name of Windows I believe.